docs/html/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Welcome to nginx!</title>
+<title>Welcome to NGINX!</title>
```

Comment:
I'd refuse to change this string: `Welcome to nginx!`. Many customers use this string as a key to indicate that NGINX is up in their automation scripts.

Reply:
Very good point - I see that even in some of our scripts, this string is case-sensitive...

Reply:
I don't have a strong opinion on this, but I feel there’s some confusion between "NGINX" and "nginx". The nginx.org website uses lowercase everywhere because the original idea was to name the product "nginx" (strictly in lowercase), and all maintainers strictly adhere to this rule. Perhaps @pluknet can correct me if I'm wrong.